### PR TITLE
Add new example:Using CarbonData to visualization in notebook

### PR DESCRIPTION
**Pull request by xubo245**
### Why is this PR needed?
 Add new example:Using CarbonData to visualization in notebook
 
 ### What changes were proposed in this PR?
Add new example:Using CarbonData to visualization in notebook
    
 ### Does this PR introduce any user interface change?
 - No

 ### Is any new testcase added?
 - No,Add new example:Using CarbonData to visualization in notebook

### CODE REVIEW

**Comment by shenjiayu17** [comment]
Build Failed  with Spark 2.4.5, Please check CI http://159.138.8.58:12602/job/ApacheCarbon_PR_Builder_2.4.5/4678/

**Comment by shenjiayu17** [comment]
Build Failed  with Spark 3.1, Please check CI http://159.138.8.58:12602/job/ApacheCarbon_PR_Builder_3.1/770/

**Comment by shenjiayu17** [comment]
Build Failed  with Spark 2.3.4, Please check CI http://159.138.8.58:12602/job/ApacheCarbonPRBuilder2.3/6386/

File: docs/using-carbondata-to-visualization_in-notebook.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+## Preparing
+It should already installed docker environment before using carbondata in notebook.
+
+## Downloading docker images of carbondata notebook 
+
+Downloading latest version of docker images of carbondata notebook 
+
+```shell
+docker pull xubo245/all-carbondata-notebook:latest
```
Comment (chenliang613) [inline docs/using-carbondata-to-visualization_in-notebook.md:9]: this is your personal folder,   is it working for all users?

Comment (xubo245) [reply docs/using-carbondata-to-visualization_in-notebook.md:9]: it's not personal folder, it's public images address of docker hub,  all user can download the carbondata images by this command if they can access docker hub official website

**Comment by chenliang613** [comment]
LGTM

**Pull request merged**